styles.css

```diff
@@ -0,0 +1,5 @@
+.local_aspiredu_lti_wrapper {
+    border: none;
+    height: 800px;
```

Comment:
Does this permanently fix our iframe to 800px?

Reply:
This is how it was before - the only difference is that I've refactored the code so it's in CSS rather than in-line style.